tests/pathfinding/test_cli.py

```diff
@@ -30,6 +30,17 @@
 }
 
 
+@pytest.fixture
+def provider_mock(monkeypatch):
+    provider_mock = Mock()
+    monkeypatch.setattr('raiden_libs.cli.HTTPProvider', provider_mock)
+    web3_mock = Mock()
+    web3_mock.return_value.net.version = 1
```

Comment:
Shouldn't this be `web3_mock.net.version.return_value` or do both work?

Reply:
This is a mock for the `Web3` class, so the return value of this mock is the `Web3` instance. So we need the `return_value` in that place. The `version` is a property, so it's not called but accessed like an attribute. So we must not use `return_value` for it.